pkg/querier/chunk_store_queryable_test.go

```diff
@@ -65,3 +65,23 @@ func mkChunk(t require.TestingT, mint, maxt model.Time, step time.Duration, enco
 	}
 	return chunk.NewChunk(userID, fp, metric, pc, mint, maxt)
 }
+
+func TestPartitionChunksOutputIsSortedByLabels(t *testing.T) {
```

Comment:
The relationship between the name and the code in this test isn't obvious to me. Perhaps add a comment to explain, or call `IsSorted()`.